BigQuery/15_convert_schema_for_labeling.sql

```diff
@@ -0,0 +1,50 @@
+SELECT 
+subject_id as id,
+row_number() over (partition by subject_id, hadm_id, stay_id order by chart_hour asc) as ICULOS,
+admittime as HospAdmTime,
+0 as Unit1,
+1 as Unit2,
+age as Age,
```

Comment:
Since this is the script for labeling, I understand that it's doing the conversion for the PNC. At some point I think we should change the code, too.